packages/components/src/components/FileHeaderInfo.js

```diff
@@ -0,0 +1,45 @@
+import { Text } from '@asyncapi/generator-react-sdk';
+
+export function FileHeaderInfo({ info, server, language }) {
+  let commentChar;
+  let line;
+
+  // Decide comment style based on language
+  if (language === 'python') {
+    commentChar = '#';
+    line = commentChar + ' ' + '/'.repeat(70);
+  } else {
+    commentChar = '//';
+    line = commentChar + '/'.repeat(70);
+  }
+

```

Comment:
please modify it in a similar way to https://github.com/asyncapi/generator/blob/master/packages/components/src/components/models.js

since now it is part of a components library, we need to make sure:
- it is easy to extend with more languages, so language maping is needed instead of such simple if/else
- proper jsdocs with `@typedef` to make sure that whoever uses such component in their code, they get nice docs rendered

example of what we got when we use Model component I referred you to

<img width="941" alt="Screenshot 2025-04-30 at 12 37 31" src="https://github.com/user-attachments/assets/88b32dad-04b2-4422-b640-3ba93f2d91cd" />